ipc/Kid.cc: add missing include of time.h

Function time() comes from <time.h>, so add the necessary include.
This fixes compilation on some configurations, for example
http://autobuild.buildroot.net/results/b33/b33e1f41e50b6e7ac3e30806b9a617d451bc27b4/

Signed-off-by: Thomas De Schampheleire <thomas.de.schampheleire@gmail.com>
Upstream-status: submitted (http://bugs.squid-cache.org/show_bug.cgi?id=3967)

---
 src/ipc/Kid.cc |  4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)


diff --git a/src/ipc/Kid.cc b/src/ipc/Kid.cc
--- a/src/ipc/Kid.cc
+++ b/src/ipc/Kid.cc
@@ -11,6 +11,10 @@
 #include <sys/wait.h>
 #endif
 
+#if HAVE_TIME_H
+#include <time.h>
+#endif
+
 int TheProcessKind = pkOther;
 
 Kid::Kid():
